torchao/quantization/prototype/mixed_precision/scripts/naive_intNwo.py

```diff
@@ -0,0 +1,40 @@
+import torch
+
+from torchao.quantization.quant_primitives import (
+    MappingType,
+    ZeroPointDomain,
+)
+
+def intN_weight_only_asym(group_size=32, n=8):
+    def apply_intN_weight_only_quant_asym(weight):
+        # avoid circular dep
+        from torchao.dtypes import to_affine_quantized
+        mapping_type = MappingType.ASYMMETRIC
```

Comment:
you can pass around this arg I think, to reduce code duplication

Reply:
Put symmetric or asymmetric into arg now

Reply:
I think Jerry means merging the two functions `apply_intN_weight_only_quant_asym` and `apply_intN_weight_only_quant_sym`, e.g. something like:

```
def apply_intN_weight_only_quant(weight: Tensor, symmetric: bool):
    # avoid circular dependency
    from torchao.dtypes import to_affine_quantized
    if symmetric:
        mapping_type = MappingType.SYMMETRIC
        quant_min = None
        quant_max = None
            ...  # other fields that are different between sym vs asym
    else:
        mapping_type = MappingType.ASYMMETRIC
        quant_min = 0
        quant_max = 2**n-1
    block_size = (1, group_size)
    eps = 1e-6
    ...  # other common fields
    return to_affine_quantized(...)
```